unleash-yggdrasil/src/strategy_grammar.pest

```diff
@@ -56,7 +56,7 @@ string_list_operation_without_case = {
     ends_with_ignore_case       = { "ends_with_any_ignore_case" }
     contains_any_ignore_case    = { "contains_any_ignore_case" }
 
-list_operation = { "in" | "not_in" }
+list_operation = { "in_ignore_case" | "not_in_ignore_case" | "in" | "not_in" }
```

Comment:
Didn't realise this when we spoke earlier but this might deserve a small redesign to the way the grammar handles these. This:

`userId in_ignore_case [1, 2, 4]`

Makes no sense and is valid right now

Up for pairing tomorrow if you want

Reply:
Nice suggestion, we fixed this